### DISCUSSION TITLE
catch all routes returns 404 in root

### DISCUSSION THREAD

**Discussion by yanv1991** [vercel/next.js#10851] [Help] [answered]
In the catch all [example](https://github.com/zeit/next.js/tree/canary/examples/catch-all-routes) you have a structure like this `post/[...slug].js` and it says it can capture urls like `post/1`, `post/2`, but if you try to access `post/` directly as a root you get 404, I have a similar scenario like this and I want to be able to handle the root `post/` with the catch all route, I tried adding an `index.js` that is exporting `[...slug].js` but the issue with this is that generates 2 separate lambdas one for `index.js` and one for `[...slug].js` as well. How can I handle this?

**Comment by Janpot** [comment, marked as answer]
relevant issue: https://github.com/zeit/next.js/issues/10488
